guarddog/analyzer/sourcecode/dll-hijacking.yml

```diff
@@ -34,6 +34,24 @@ rules:
                 # a string with .dll or .so in it
                 - pattern: "..."
                 - pattern-regex: (?i).*?\.(dll|so)
+        - patterns:
+          - pattern: "..."
+          - pattern-either:
+            # See MITRE ATT&CK "System Binary Proxy Execution" techniques
+            # https://attack.mitre.org/techniques/T1218/
+            - pattern-regex: control(.exe)?\s*\S*.cpl
```

Comment:
\s+\S+ meaning that we need at least one space and one nonspace

Reply:
Changed